adapters/handlers/rest/configure_api.go

```diff
@@ -199,7 +200,7 @@ func MakeAppState(ctx context.Context, options *swag.CommandLineOptionsGroup) *s
 	appState := startupRoutine(ctx, options)
 
 	if appState.ServerConfig.Config.Monitoring.Enabled {
-		appState.ServerMetrics = monitoring.NewServerMetrics(appState.ServerConfig.Config.Monitoring.MetricsNamespace, prometheus.DefaultRegisterer)
+		appState.ServerMetrics = monitoring.NewServerMetrics(monitoring.DefaultMetricsNamespace, prometheus.DefaultRegisterer)
```

Comment:
Why this change? It can break something it the configured value is different

Reply:
I added this metric recently. Not too much worried about breaking. Plus we have flexible breaking policy for metrics.
https://weaviate.io/developers/weaviate/configuration/monitoring#versioning

> Metrics, however, have shorter feature lifecycles. It can sometimes be necessary to introduce an incompatible change or entirely remove a metric, for example, because the cost of observing a specific metric in production has grown too high. As a result, it is possible that a Weaviate minor release contains a breaking change for the Monitoring system. If so, it will be clearly highlighted in the release notes.